src/assets/HTMLAsset.js

```diff
@@ -64,7 +64,8 @@ const META = {
 const SCRIPT_TYPES = {
   'application/javascript': 'js',
   'text/javascript': 'js',
-  'application/json': false
+  'application/json': false,
+  'text/html': false
```

Comment:
Maybe we should change it to a whitelist instead of a blacklist? Basically, just `application/javascript` and  `text/javascript`, or if the type attribute is not specified. We could also enable typescript, coffeescript, etc.

Reply:
That makes sense. In the case of no `type` attribute you would like to keep existing behavior and default to JS? Wondering if this will be a breaking change for existing users.